baseapp/baseapp_test.go

```diff
@@ -491,16 +494,50 @@ func TestInfo(t *testing.T) {
 
 	// ----- test an empty response -------
 	reqInfo := abci.RequestInfo{}
-	res := app.Info(reqInfo)
+
+	//res := app.Info(reqInfo)
 
 	// should be empty
-	assert.Equal(t, "", res.Version)
-	assert.Equal(t, t.Name(), res.GetData())
-	assert.Equal(t, int64(0), res.LastBlockHeight)
-	require.Equal(t, []uint8(nil), res.LastBlockAppHash)
+
+	//assert.Equal(t, "", res.Version)
+	//assert.Equal(t, t.Name(), res.GetData())
+	//assert.Equal(t, int64(0), res.LastBlockHeight)
+	//require.Equal(t, []uint8(nil), res.LastBlockAppHash)
```

Comment:
this is checked below, so maybe we can remove these comments?

Reply:
yeah